src/PhpSpreadsheet/Reader/Xlsx.php

```diff
@@ -1357,63 +1357,67 @@ public function load($pFilename)
                                         'SimpleXMLElement',
                                         Settings::getLibXmlLoaderOptions()
                                     );
-                                    $vmlCommentsFile->registerXPathNamespace('v', 'urn:schemas-microsoft-com:vml');
-
-                                    $shapes = $vmlCommentsFile->xpath('//v:shape');
-                                    foreach ($shapes as $shape) {
-                                        $shape->registerXPathNamespace('v', 'urn:schemas-microsoft-com:vml');
-
-                                        if (isset($shape['style'])) {
-                                            $style = (string) $shape['style'];
-                                            $fillColor = strtoupper(substr((string) $shape['fillcolor'], 1));
-                                            $column = null;
-                                            $row = null;
-
-                                            $clientData = $shape->xpath('.//x:ClientData');
-                                            if (is_array($clientData) && !empty($clientData)) {
-                                                $clientData = $clientData[0];
-
-                                                if (isset($clientData['ObjectType']) && (string) $clientData['ObjectType'] == 'Note') {
-                                                    $temp = $clientData->xpath('.//x:Row');
-                                                    if (is_array($temp)) {
-                                                        $row = $temp[0];
-                                                    }
 
-                                                    $temp = $clientData->xpath('.//x:Column');
-                                                    if (is_array($temp)) {
-                                                        $column = $temp[0];
+                                    if($vmlCommentsFile !== FALSE) {
```

Comment:
Instead of adding one more level of nested block, it should be extracted as a private method.